### DISCUSSION TITLE
imageMath.py: Command not found.

### DISCUSSION THREAD

**Discussion by PriyomRoy** [alexisInSAR/EZ-InSAR#46] [Help]
At SBAS interferogram step of STAMPS

Looping over the interferogram list now:
20230124 20230205
imageMath.py -e='a*conj(b)' --a=/home/gsg/Joshimath/cropped/SLC/20230124/20230124.slc.full --b=/home/gsg/Joshimath/cropped/SLC/20230205/20230205.slc.full -o /home/gsg/Joshimath/stackStaMPS/INSAR_20230313/SMALL_BASELINES/20230124_20230205/isce_minrefdem.int -t CFLOAT -s BIP
imageMath.py: Command not found.

**Comment by alexisInSAR** [comment]
This command is from the ISCE2 package. It seems that the conda environment is not loaded. Can you try this command in your Linux terminal to check its availability?
